base/setup/usetup/muilanguages.h

```diff
@@ -242,9 +242,9 @@
 // #ifdef LANGUAGE_HI_IN
     // #include "lang/hi-IN.h"
 // #endif
-// #ifdef LANGUAGE_HR_HR
-    // #include "lang/hr-HR.h"
-// #endif
+ #ifdef LANGUAGE_HR_HR
+    #include "lang/hr-HR.h"
+ #endif
```

Comment:
Indentation is a bit off
```suggestion
#ifdef LANGUAGE_HR_HR
    #include "lang/hr-HR.h"
#endif
```